packages/ses/package.json

```diff
@@ -32,6 +32,7 @@
     "qt": "tap --no-esm --no-coverage --reporter spec 'test/**/*.test.js'",
     "test": "yarn build && yarn qt",
     "test262": "tap --no-esm --no-coverage --reporter spec test262/*.js",
+    "test:platform-compatability": "node test/package/test.cjs",
```

Comment:
I don’t think we need this.

Reply:
@kriskowal are you saying I should remove the npm script and just call this directly in the ci?

Reply:
I retract my comment. We clearly do need this for your changes to CI. Thanks for making that connection clear!